README.md

```diff
@@ -56,6 +56,41 @@ $ oc get deploy -nopenshift-compliance
 $ oc get pods -nopenshift-compliance
 ```
 
+### Deploying with Helm
+
+The repository contains a [Helm](https://helm.sh/) chart that deploys the
+compliance-operator. This chart is currently not published to any official
+registries. You're required to run the chart from this repository.
+
+Make sure you create the namespace prior to running `helm install`:
+
+```
+$ kubectl create namespace openshift-compliance
```

Comment:
Would be better as a `create -f deploy/ns.yaml` so they don't miss out on namespace annotations.